docs/content/packaging/reference.md

```diff
@@ -83,6 +83,28 @@ Additionally, Hermit will create several synthetic channels which are checked fo
 
 This allows projects to pin to stable releases.
 
+## Dependencies
+
+Hermit supports two kinds of dependencies between packages, direct dependencies and runtime dependencies.
+
+### Direct dependencies
+
+Direct dependencies are declared in the manifest files using a `requires` definition in the manifest file. 
+This lists the packages that are needed in the environment to use the given package. 
+The package references in the `requires` list can either refer to an explicit package, or to a value defined in the `provides` definition of the dependency.
+For example, `requires = ["jre"]` would work with any package defining `provides = ["jre"]` in its definition.
+
+When a package with `requires` definition is installed, all its dependencies are installed to the target environment as well.
+
+### Runtime dependencies
+
+Runtime dependencies are package dependencies that are not installed into the target environment.
+Instead, they are injected to the environment variables if the package binary at the execution time.
```

Comment:
The paragraph below seems to better describe it, than this second sentence. Maybe just remove this one?